ohapi/api.py

```diff
@@ -200,3 +191,20 @@ def message(subject, message, access_token, all_members=False,
                'project_member_ids': project_member_ids,
                'subject': subject,
                'message': message})
+
+
+def exceeds_size(filesize, max_bytes, target_filepath):
+    if filesize > max_bytes:
+        logging.info('Skipping {}, {} > {}'.format(
+            target_filepath, format_size(filesize), format_size(max_bytes)))
+        return True
+    return False
+
+
+def process_info(remote_file_info, filesize, target_filepath):
```

Comment:
Why does this function need an empty `return` statement that will just yield `None`?

Reply:
Yeah maybe it isn't needed there. I added it just to prevent any boundary-condition mistakes. Will remove that. Thanks :)

Reply:
I think just removing is not what's wanted here. 

```
if remote_size == filesize:
+        logging.info('Skipping {}, remote exists with matching name and '
+                     'file size'.format(target_filepath))
```

You actually want to stop uploading a file if this is true. So you will need to return something (e.g. `False`) and then do a test for the return-value of `process_info` in the `upload_file` function to make sure that it will be skipped, analogous to the `exceeds_size` function. Eg

```
if process_info(…) is False:
    return
```